spec/components/related_navigation_spec.rb

```diff
@@ -158,6 +158,28 @@ def construct_links(type, base_path, title, document_type = nil, locale = nil)
     assert_select ".gem-c-related-navigation__nav-section[aria-labelledby]"
   end
 
+  it "adds a show more toggle link to long sections for related content" do
+    content_item = { "links" => { "ordered_related_items" => [] } }
+    [
+      { title: "Find an apprenticeship", base_path: "/apply-apprenticeship" },
+      { title: "Care to Learn", base_path: "/care-to-learn" },
+      { title: "Career skills and training", base_path: "/career-skills-and-training" },
+      { title: "Learner Support", base_path: "/learner-support" },
+      { title: "Offer a trainee work experience", base_path: "/employ-trainees" },
+      { title: "What qualification levels mean", base_path: "/what-different-qualification-levels-mean" },
+      { title: "Find a traineeship", base_path: "/find-traineeship" },
+    ].each do |item|
+      content_item["links"]["ordered_related_items"] << { "title" => item[:title], "base_path" => item[:base_path] }
+    end
+
+    render_component(content_item: content_item)
+
+    assert_select ".gem-c-related-navigation__section-link[href=\"/apply-apprenticeship\"]", text: "Find an apprenticeship"
+    assert_select ".gem-c-related-navigation__toggle", text: /\+\s2 more[.\n+\s]+related content/
```

Comment:
I have some reservations about the grammar here. This works fine for "two more world locations" but gets a bit weird when it comes to "two more related content". I'd suggest you run it by Anika to see if this is good enough or whether we want to expand these to "two more related content list items" or something similar.